artifactory/commands/python/python.go

```diff
@@ -58,6 +59,9 @@ func (pc *PythonCommand) Run() (err error) {
 		// Need to collect build info
```

Comment:
In line 38 9n this file, let's replace - 
```
	if err != nil {
		return err
	}
```
with - 
```
	if err != nil {
		return
	}
```

Also, at the botton of this function, the following section - 
```
		err = errorutils.CheckError(pythonModule.RunInstallAndCollectDependencies(pc.args))
		if err != nil {
			return
		}
	} else {
		// Python native command
		err = gofrogcmd.RunCmd(pc)
		if err != nil {
			return
		}
	}
	return
```
can be simplified to - 
```
		err = errorutils.CheckError(pythonModule.RunInstallAndCollectDependencies(pc.args))
	} else {
		// Python native command
		err = gofrogcmd.RunCmd(pc)
	}
	return
```